### ISSUE TITLE
Generate list of potential data sources

### ISSUE THREAD

**Issue by CubeOfCheese** [CubeOfCheese/good-storey#2]
Should check with Cosmo or Billy T. for verification that data can be accessed. The sources should be categorized based on ease of access.

**Comment by amtims** [comment]
Hey team, here is a list of websites to gather sources and data, ordered from best to least helpful. Depending on the mechanism used to rate the products, Certified B Corp seems to be the most helpful as it provides sources, requires certifications and is unbiased. The other websites included might be helpful if we need to find products/companies that are not B-Certified, but may lack sources. 

Let me know if there is a different preferred format to list these, and if Billy and/or Cosmo can verify the usability of these sites to source data, thanks! 

1. Certified B Corp:
- [Directory](https://bcorporation.net/directory)
- Example: [Patagonia](https://bcorporation.net/directory/patagonia-inc)
- Patagonia [Impact Report](https://www.patagonia.com/on/demandware.static/-/Library-Sites-PatagoniaShared/default/dw9e73b6b5/PDF-US/2017-BCORP-pages_022218.pdf) 2017
- Article [example](https://bthechange.com/best-for-the-world-2018-all-honorees-f30a880f8ac0)


2. Good On You: Has a rating system, but it's broad, and no primary sources available about specific data sources. Seems to have customers "take their word" on other company ratings
- Example: [MUD Jeans](https://directory.goodonyou.eco/brand/mud-jeans)

3. Shop Ethical: Addresses Praise vs. Criticism on companies. Does not numerically rate, but may help if it is a company that isn't certified through B-Corp

- Example: [Fairphone](https://guide.ethical.org.au/company/?company=5909)

4. Good Shopping Guide: Rates but doesn't provide data sources
- Website: [Ethical Shopping](https://thegoodshoppingguide.com/)
- Example: [Coffee Brands](https://thegoodshoppingguide.com/subject/coffee-shops/)

5. EthicalConsumer (requires a $30/year subscription but might help to score some local companies)
- [Website](https://www.ethicalconsumer.org/)

**Comment by CubeOfCheese** [comment]
These are all good and I see us using these in the future, but for this first week I think we need to find more raw data. It is easy for a normal person to type a company name into the search bar for the b corp directory but not so easy to access that same data programmatically for example. I'll start looking too. Good work!

**Comment by CubeOfCheese** [comment]
I just found this API that listed all of its sources. We'll probably be able to find some sources here. I haven't gotten to look through them yet.
https://www.csrhub.com/our_data_sources?_ga=2.33189245.234147823.1589855740-1428106147.1589855740&page=1

**Comment by amtims** [comment]
[This page](https://www.csrhub.com/subscription_sample/) has a sample of Walmart data, but to use other data we might need to buy a subscription... I'm trying to find out more in the meantime.

**Comment by CubeOfCheese** [comment]
I haven't found a super easy to use API yet so I think looking at government data might be a good next step. I'm not sure what government data would indicate ethics or where to look for it yet.

**Comment by amtims** [comment]
Here's an API for [GoodGuide](https://www.programmableweb.com/api/goodguide-rest-api), which operates a similar service. This is a little outside my understanding of coding so I'm not sure whether this is helpful, but thought I'd post it anyway. There is also an API directory which could help as well.

**Comment by CubeOfCheese** [comment]
I came across that api too, but if you click the link, it's broken so I think gud guide used to provide an API but discontinued it.

**Comment by amtims** [comment]
I saw the broken links, but then clicked further and found the website working. Do [this](https://www.goodguide.com/categories/152675-conditioner-reviews-and-ratings#/) or [this](https://www.goodguide.com/#/) work? Or is that just the website and not helpful to anything related to the use of the API?

**Comment by amtims** [comment]
Oh I see now that it wouldn't work. I'll keep looking!

**Comment by CubeOfCheese** [comment]
Yeah those are just the website unfortunately. If we really can't find anything after further searching we might have to come up with a hacky way of accessing the data, but that would take a lot more work. Look around the websites you find (specifically in the footer) for either the word "API" or "developers".

**Comment by CubeOfCheese** [comment]
I'm going to look through [this](https://any-api.com/). Hopefully I find something

**Comment by CubeOfCheese** [comment]
@amtims Can you check this source out? Let me know what you think
https://www.bluesign.com/downloads/general/reference-list/bluesign-reference-list.pdf

**Comment by CubeOfCheese** [comment]
Also there is a list of sources at the bottom of this pdf from good on you.
https://goodonyou.eco/wp-content/uploads/2020/01/Good-On-You-Brand-Rating-System_Jan-2020.pdf

**Comment by amtims** [comment]
These are both great resources for the textile industry standards and are reputable sources with transparency about where the data comes from!

**Comment by CubeOfCheese** [comment]
B Corporation's Terms of Use prohibit the use of web spiders and they do not have an API available, so there is no way to access their data for use in our product without making an official agreement with them (I imagine this would cost a lot of money).

**Comment by CubeOfCheese** [comment]
https://www.bbb.org

**Comment by CubeOfCheese** [comment]
https://data.cdp.net/Companies/2014-Forest-Risk-Assessment-by-Sector/t6pz-w5be

**Comment by CubeOfCheese** [comment]
https://www.fec.gov/data/committee/C00063164/?tab=filings

**Comment by CubeOfCheese** [comment]
https://guide.ethical.org.au/company/?company=1646
Shop ethical lists sources at the bottom of their page.

**Comment by CubeOfCheese** [comment]
UK living wage: https://www.livingwage.org.uk/accredited-living-wage-employers

**Comment by CubeOfCheese** [comment]
Tree planting: https://ecologi.com/leaderboard/business

**Comment by CubeOfCheese** [comment]
Here's a list of standards: https://benefitcorp.net/how-do-i-pick-third-party-standard

**Comment by CubeOfCheese** [comment]
This is probably the most comprehensive directory of public benefit corporations available: https://benefitcorp.net/businesses/find-a-benefit-corp

**Comment by CubeOfCheese** [comment]
https://www.ethicalconsumer.org/ethical-consumer-magazine

**Comment by CubeOfCheese** [comment]
https://www.grabyourwallet.org/the-list

**Comment by CubeOfCheese** [comment]
This is crowdsourced data so probably not good to be used in Ethicli, but maybe something we could share: https://foundation.mozilla.org/es/privacynotincluded/

**Comment by amtims** [comment]
[[Green Sports Alliance](https://greensportsalliance.org/all-members/)] (NBA is a part of)

**Comment by CubeOfCheese** [comment]
Companies who have made commitments to renewable energy plans https://www.there100.org/re100-members
